class AnError(Exception):
    pass


raise AnError()
